gradle-plugins/src/main/kotlin/io.opentelemetry.instrumentation.muzzle-check.gradle.kts

```diff
@@ -85,10 +85,14 @@ tasks.withType<ShadowJar>().configureEach {
   // rewrite dependencies calling Logger.getLogger
   relocate("java.util.logging.Logger", "io.opentelemetry.javaagent.bootstrap.PatchLogger")
 
-  // prevents conflict with library instrumentation
-  relocate("io.opentelemetry.instrumentation", "io.opentelemetry.javaagent.shaded.instrumentation")
+  // prevents conflict with library instrumentation, since these classes live in the bootstrap class loader
+  relocate("io.opentelemetry.instrumentation", "io.opentelemetry.javaagent.shaded.instrumentation") {
+    // Exclude resource providers since they live in the agent class loader
+    exclude("io.opentelemetry.instrumentation.resources.*")
+    exclude("io.opentelemetry.instrumentation.spring.resources.*")
```

Comment:
@jack-berg these two lines were the fix, I forgot that the distro does its own shading, so had to copy over the new shading exclusions you added to here also

Reply:
Is that different from what I committed [here](https://github.com/open-telemetry/opentelemetry-java-instrumentation/pull/7136/commits/609b650a4b071f9d5d0a741215d08eafed1b3ec1)?

Reply:
ya, the shading is duplicated in 3 places in this repository 😅

Reply:
Oh wait, [this is the one](https://github.com/open-telemetry/opentelemetry-java-instrumentation/pull/7136/files#diff-32a60635ea004e3a89bfb8a93adba9e6cf0b22cb4c325ce99edf5cf0a12f4832R8-R9) I missed. Thanks!